tests/pipelines/pixart_alpha/test_pixart.py

```diff
@@ -341,14 +347,13 @@ def test_pixart_1024(self):
         image = pipe(prompt, generator=generator, output_type="np").images
 
         image_slice = image[0, -3:, -3:, -1]
+        expected_slice = np.array([0.2891, 0.2749, 0.2595, 0.3020, 0.2698, 0.2671, 0.3169, 0.2993, 0.3179])
```

Comment:
That's quite big a change. Why is this coming?

Reply:
The difference produced currently between the output and expected output is quite large
https://github.com/huggingface/diffusers/actions/runs/7876381153/job/21491555064#step:7:485

I just ran `print_tensor_test` on the current outputs in our new containers (with torch 2.2) and updated. If this seems like a deeper issue, let me know how you want to tackle it.

Reply:
Actually, the values will have to change here. This test is running 20 inference steps, which is too many.

Reply:
But we're not changing the `num_inference_steps` yet no? It's okay if the values are changed because of a reduced number of steps. But at 20 steps, the assertion values shouldn't change.

Reply:
The tests are failing with torch 2.1 as well. It could also be the CUDA version in the runners.  I've run the pipeline example in the docs and the generated image seems fine. 
![pixart](https://github.com/huggingface/diffusers/assets/7529846/3318ef77-76c5-4ec3-88b8-6ef4086d7f29)
 
It feels like the test is most likely failing due to changes in torch version or CUDA version. But if you would like to work on an alternative solution, feel free to take a look.

Reply:
Okay let's merge then.